src/components/Dropdown/Dropdown.tsx

```diff
@@ -60,6 +60,7 @@ export const Dropdown = ({
   getPopupContainer,
   initialSelectedItems = defaults.initialSelectedItems,
   multiple,
+  selectable = true,
```

Comment:
what is the purpose of this "selectable" prop? I don't recall a dropdown where items are not selectable

Reply:
this applies to dropdowns that for example open another element on element click, so the item should not stay selected / highlighted

Reply:
ok clear, let's change the name since it's a bool,  like "shouldHighlightSelection" or something similar

Reply:
Fair enough to consider a verb-like word since it is a bool. I'd suggest `persistSelection` as the prop not only would allow the highlight of previous choices but it works also on persisting values previously selected. Let me know if you like that.